### DISCUSSION TITLE
How to add assets to my extension (module)?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17647] [Archive]
I am writing a module extension and I was wondering how I could integrate files …

On the one hand I need it to display images via the `<img>` tag and on the other hand I want to provide a download as a .zip file.

In my module, i have created an assets folder, but it is not copied to the `dist` folder when `build`ing the extension …

How can I do this?

**Comment by ghost** [comment]
Any ideas?

**Comment by nickrum** [comment]
If you want to provide the assets as a downloadable zip file, the easiest would probably be to create an endpoint extension that serves the assets to your module.
